.vscode/bevy.code-snippets

```diff
@@ -0,0 +1,75 @@
+{
+    "Bevy: New module-level function Plugin": {
+        "scope": "rust",
+        "prefix": "plugin_fn",
```

Comment:
```suggestion
        "prefix": "plugin",
```
works well enough in my experience.

Reply:
We might want a struct Plugin snippet at some point?

Reply:
Too niche imo

Reply:
Follow-up: I'd prefer to have the more common use-case easy to use (`plugin`) and have the more specialized one be more verbose (`plugin_struct`)